pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator.go

```diff
@@ -97,6 +97,18 @@ func (mutator *VMIsMutator) Mutate(ar *admissionv1.AdmissionReview) *admissionv1
 			addNodeSelector(newVMI, v1.SEVESLabel)
 		}
 
+		if newVMI.Spec.Domain.CPU.IsolateEmulatorThread {
```

Comment:
Missing dedicated cpu check.

Reply:
We discussed it in the original PR, there is currently a validating webhook preventing the user from setting `IsolateEmulatorThread` without `dedicatedCPUs`. This is why in the code only IsolateEmulatorThread is checked.
However, I think that if we want to fix it we need to do it on a separate PR.

Reply:
You are right, thanks for the reminder.